op-supervisor/supervisor/backend/l1access/l1_accessor.go

```diff
@@ -159,7 +159,26 @@ func (p *L1Accessor) onFinalized(ctx context.Context, ref eth.L1BlockRef) {
 }
 
 func (p *L1Accessor) onLatest(ctx context.Context, ref eth.L1BlockRef) {
-	p.tipHeight = ref.Number
+	// Stop if the block is the same or older than the tip
+	if ref.Number == p.tip.Number && ref.Hash == p.tip.Hash {
```

Comment:
what are the chances that the hashes match but the numbers dont?

Reply:
Basically 0, but it's not really necessary to check.